[2024-08-16T11:13:13.105+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T11:13:09.075750+00:00 [queued]>
[2024-08-16T11:13:13.120+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T11:13:09.075750+00:00 [queued]>
[2024-08-16T11:13:13.121+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-08-16T11:13:13.159+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): compare_data2threshold_task> on 2024-08-16 11:13:09.075750+00:00
[2024-08-16T11:13:13.167+0000] {standard_task_runner.py:60} INFO - Started process 8052 to run task
[2024-08-16T11:13:13.174+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'compare_data2threshold', 'compare_data2threshold_task', 'manual__2024-08-16T11:13:09.075750+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/compare_data2threshold.py', '--cfg-path', '/tmp/tmp00xnzxde']
[2024-08-16T11:13:13.176+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask compare_data2threshold_task
[2024-08-16T11:13:13.198+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-08-16T11:13:13.258+0000] {task_command.py:423} INFO - Running <TaskInstance: compare_data2threshold.compare_data2threshold_task manual__2024-08-16T11:13:09.075750+00:00 [running]> on host 282fa7a8e0f3
[2024-08-16T11:13:13.406+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='compare_data2threshold' AIRFLOW_CTX_TASK_ID='compare_data2threshold_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-16T11:13:09.075750+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-16T11:13:09.075750+00:00'
[2024-08-16T11:13:13.409+0000] {compare_data2threshold.py:36} INFO - unit_threshold : 120
[2024-08-16T11:13:13.409+0000] {compare_data2threshold.py:37} INFO - global_threshold : 2000
[2024-08-16T11:13:13.458+0000] {compare_data2threshold.py:54} INFO - Total des counts : 1672
[2024-08-16T11:13:13.459+0000] {compare_data2threshold.py:60} INFO - Le total 1672 ne d√©passe pas le seuil global 2000
[2024-08-16T11:13:13.460+0000] {compare_data2threshold.py:65} INFO - Au moins une ligne est au-dessus de unit_threshold
[2024-08-16T11:13:13.461+0000] {compare_data2threshold.py:70} INFO - Nous devons reentrainer le model avec les nouvelles data
[2024-08-16T11:13:13.501+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-08-16T11:13:13.539+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=compare_data2threshold, task_id=compare_data2threshold_task, execution_date=20240816T111309, start_date=20240816T111313, end_date=20240816T111313
[2024-08-16T11:13:13.631+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-16T11:13:13.745+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
